docs/development/developer_guide.md

```diff
@@ -47,11 +47,16 @@ a K8s cluster. Set your environment:
 
 ```sh
 export KUBECONFIG=$(echo ~/.kube/config)
-export KUBEFLOW_NAMESPACE=$(your_namespace)
+export KUBEFLOW_NAMESPACE=trainingoperator
```

Comment:
```suggestion
```

The example PyTochJob uses the kubeflow namespace, and the namespace is set up when we install the training-operator. So, we could remove this line.